buildspec-windows-cmake.yml

```diff
@@ -9,7 +9,9 @@ phases:
   install:
     commands:
       - choco install -y --no-progress cmake --installargs 'ADD_CMAKE_TO_PATH=System'
-      - choco install -y --no-progress winflexbison3 ninja
+      - choco install cyg-get -y --no-progress
```

Comment:
Something about installing one package manager with another package manager so the managed package manager can manage some packages is hurting my brain.